docs/user-guide.md

```diff
@@ -1,15 +1,53 @@
 ## User Guide
 
 ### Installation
+Version 1.0.0 is very experimental and has some known bugs, it is highly recommended to install nixd from source.
```

Comment:
How about just do not mention `1.0.0` because we may release something like `1.0.1` or whatever.

```diff
- Version 1.0.0 is very experimental and has some known bugs, it is highly recommended to install nixd from source.
+ At this time (2023-06-14), nixd is under rapid development and it is highly recommended to install nixd from source.
```